README.md

```diff
@@ -93,195 +93,38 @@ Mapbox has a free tier, however there are paid options available. For most users
-
-   Find the configuration file by running `sudo -u postgres psql -c "SHOW hba_file;"`. Edit the first uncommented line of that file to be
-      ```
-      local   all             postgres                                trust
-      ```
-   Then, add the following lines at the bottom:
-      ```
-      # Allow any user on the local system using local loopback TCP/IP connections
-      # to connect to any database with any database user name.
-      host    all             all             127.0.0.1/32            trust
-      # The same over IPv6.
-      host    all             all             ::1/128                 trust
-      ```
-   Finally, run `sudo service postgresql restart` for the changes to take effect.    ​
-
-4.  Enter psql client to check server success. If it works, then you can exit psql.
-
-    - macOS: `psql postgres`
-    - Ubuntu: `psql -U postgres`
-
-5.  Create and upgrade the local postgres database as well as populate the Data Catalog tables: `./scripts/db/postgres/dev/init_db.py --populate_indicators`
-
-    This command will create a database for each deployment defined in config, you can optionally run the command for just one deployment like `./scripts/db/postgres/dev/init_db.py <ZEN_ENV> --populate_indicators`. Also you can run just `./scripts/db/postgres/dev/init_db.py` to only upgrade the database and not populate Data Catalog.
-
-6. Create an admin user for your local web app like below.
-
-      ```
-      ./scripts/create_user.py -f <first name> -l <last name> -u <email> -p <password> -a -d postgresql://postgres:@localhost/{ZEN_ENV}-local
-      ```
-
-### Hasura
-
-Start Hasura: `./scripts/db/hasura/dev/start_hasura.sh <ZEN_ENV>-local`
+### Run Web Server
 
-### Run webpack locally
+1. Start your development environment: `DEFAULT_DRUID_HOST=<public production Druid server IP> ZEN_ENV=<specify environment> docker compose up` (you need to specify druid host and environment)
+2. In a seperate terminate, create user account `docker compose exec web /bin/bash -c "source venv/bin/activate && ./scripts/create_user.py --u me@mydomain.com -p admin --first_name=admin --last_name=istrator -a"`
+3. Browse to website on [http://localhost:5000](http://localhost:5000) and log in with the credentials used in step 5.
 
-Run command: `node_modules/.bin/webpack-cli serve --config web/webpack.config.js --mode 'development'`
+### Run Pipeline
 
-### Run web server locally
+1. Execute the pipeline `DEFAULT_DRUID_HOST=http://aws-druid.corp.clambda.com COMMAND="./pipeline/br/generate/zeus_generate run" docker compose --profile=pipeline up pipeline`
```

Comment:
Ok - I see zeus_process is setting `export ZEN_ENV='br'` - so that's fine ; I'm not sure I follow the DATABASE_URL flow exactly, I'll remove `DATABASE_URL` and `ZEN_ENV` from the pipeline config in `docker-compose.yml` and test.

Reply:
I was missing PYTHONPATH environment variable, so I've added that.